frontend/src/components/label/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled'
+import tag from 'clean-tag'
+import {
+  fontSize,
+  fontWeight,
+  lineHeight,
+  space,
+  color,
+  display,
+  border,
+  borderColor,
+  width,
+  height,
+  position,
+  zIndex,
+  top,
+  right,
+  bottom,
+  left,
+} from 'styled-system'
+
+export const Label = styled(tag.label)`
+  ${fontSize};
+  ${fontWeight};
+  ${lineHeight};
+  ${space};
+  ${color};
+  ${display};
+  ${border};
+  ${borderColor};
+  ${width};
+  ${height};
+  ${position};
+  ${zIndex};
+  ${top};
+  ${right};
+  ${bottom};
+  ${left};
+`
```

Comment:
What about adding some defaults?

Reply:
I'm honestly not sure what our defaults would look like right now! I figure we can add them as use cases come up?